dashboard/config/locales/short_instructions.en.yml

```diff
@@ -1724,9 +1724,6 @@ en:
       HOC 2018 Level_5: Explore the shipwreck to reach the chest.
       HOC 2018 Level_6: Navigate through the ice to reach the salmon.
       HOC 2018 Level_7a: Search these underwater ruins for a chest.
-      HOC 2018 Level_8: 'TEST SHORT
-
-'
       HOC 2018 Level_9b: 'Use the coral as waypoints to reach the treasure.
 
 '
```

Comment:
@Hamms This is unrelated to the sync,  I have noticed a few of these opening quote mark in parts of the document.  Is this intentional?

Reply:
in this case, it's actually a closing quote; the `HOC 2018 Level_8` string is `TEST SHORT\n\n`, and this is just how yaml is choosing to display it.

I'm not sure I would strictly call it _intentional_, but it's certainly not a problem